packages/components/atoms/f-form-field/src/components/FormField.vue

```diff
@@ -17,14 +17,19 @@
                 :is-inline="isInline"
                 :data-test-id="testId.label">
                 {{ labelText }}
+                <template #description>
+                    <p
```

Comment:
Does this render as a `p` tag inside a label? If so, this would be better as a span, as block-level elements like `<p>` inside a label isn't valid html.